src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -406,6 +406,16 @@ class genUtils {
         return inputNumber.toLocaleString(undefined, { minimumFractionDigits: n, maximumFractionDigits: n });
     };
 
+    /***  File Methods ***/
+    
+    static isFileOfType(fileName: string, possibleFileTypes: string[]): boolean {
+        const fileParts = _.split(fileName,".");
+        const fileExtension = fileParts[fileParts.length-1];
+
+        return fileParts.length === 2 &&
```

Comment:
this is wrong. What if file contains dots like: ravendb.net.backup? In general you should check if path ends with "." + extension.

Reply:
done - using getFileExtension method